cmd/queue/main_test.go

```diff
@@ -443,6 +443,38 @@ func TestQueueTraceSpans(t *testing.T) {
 	}
 }
 
+func BenchmarkProxyHandler(b *testing.B) {
+	var httpHandler http.HandlerFunc = func(w http.ResponseWriter, r *http.Request) {}
+	server := httptest.NewServer(httpHandler)
+	serverURL, _ := url.Parse(server.URL)
+	defer server.Close()
+	proxy := httputil.NewSingleHostReverseProxy(serverURL)
+	params := queue.BreakerParams{QueueDepth: 10, MaxConcurrency: 10, InitialCapacity: 10}
+	breaker := queue.NewBreaker(params)
+	testFunc := func() {
+		reqChan := make(chan queue.ReqEvent, requestCountingQueueLength)
+		h := proxyHandler(reqChan, breaker, true /*tracingEnabled*/, proxy)
```

Comment:
We probably don't create proxy on every request, so we shouldn't do it here either.
I wonder how many of the other handlers we want to include (or not).
/cc @markusthoemmes

Reply:
Yeah I agree, this should be globally allocated.